packages/safe-tools-client/public/images/icons/card-cpxd-token.svg

```diff
@@ -0,0 +1 @@
+<svg height="41" viewBox="0 0 41 41" width="41" xmlns="http://www.w3.org/2000/svg"><path d="m20 90a20 20 0 1 1 -20 20 20 20 0 0 1 20-20z" fill="#fff" stroke="#707070" transform="translate(.5 -89.5)"/><g fill="#00ebe5"><path d="m77.169 332.785 4.966-2.476-11.363-5.665-4.935 2.46z" transform="translate(-56.669 -300.23)"/><path d="m83.521 305.576 4.948-2.467-17.763-8.857-4.948 2.466z" transform="translate(-56.607 -276.219)"/><path d="m131.652 299.189 4.952-2.47-4.948-2.466-4.952 2.469z" transform="translate(-104.756 -276.219)"/></g><path d="m70.615 263.823-4.951 2.468 11.367 5.668 4.952-2.47z" fill="#37eb77" transform="translate(-56.532 -252.179)"/><path d="m131.656 268.767 4.968-2.476-4.951-2.468-4.973 2.477z" fill="#37eb77" transform="translate(-104.756 -252.179)"/><path d="m101.112 248.583-4.968 2.477 4.968 2.476 4.968-2.476z" fill="#37eb77" transform="translate(-80.613 -240.138)"/></svg>
```

Comment:
Could these token icons be in `ember-shared`? Maybe in a `tokens` directory

Reply:
Possibly. I think this will shift a little with my upcoming tokenlist work, so I'm going to leave it for now.